uw-frame-components/portal/features/controllers.js

```diff
@@ -67,10 +67,12 @@ define(['angular','require'], function(angular, require) {
       //local functions ---------------------------------------------------------
 
       var getPopups = function(){
-        if(!$rootScope.GuestMode) {
+        if (!$rootScope.GuestMode) {
           portalFeaturesService.getUnseenPopups().then(function(unseenPopups) {
-            if(unseenPopups.length !=0 && !$rootScope.GuestMode){
+            if (unseenPopups.length !=0 && !$rootScope.GuestMode) {
+              console.log(unseenPopups);
               var orderedPopups = $filter('orderBy')(unseenPopups, ['popup.startYear', 'popup.startMonth', 'popup.startDay', 'id']);
+              console.log(orderedPopups);
```

Comment:
Suggest wrapping console.log in a `if(APP_FLAGS.debug)` or use [`$log`](https://docs.angularjs.org/api/ng/service/$log).